server/config.py

```diff
@@ -1,3 +1,7 @@
 import os
 
 TEAM_NAME = os.environ['TEAM_NAME']
+
+class Config(object):
+    SECRET_KEY = 'I am cool as superhero'
+    MONGODB_HOST = "mongodb+srv://TeamBLT-Admin:jcPCzsMV2021HatchwaysTeamBLT@teambltcluster0.5hi8c.mongodb.net/test?retryWrites=true&w=majority"
```

Comment:
Remove this and change the password. If you run your app with `pipenv run flask run`, it will load the data from the .env and you will be able to access these fields using os.environ, as we do above with TEAM_NAME